.github/workflows/linters.yml

```diff
@@ -8,25 +8,41 @@ name: Format code using black and isort
 
 on:
   push:
-    branches: [ "develop", "issue23-precommithooks" ]
-
+    branches-ignore:
+          - master
 jobs:
   linters:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Format code with black 
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install dependencies
         run: |
-          pip install black==21.12b0
-          black .
-      - name: Format code with isort
+          python -m pip install --upgrade pip
+          python -m pip install tomli
```

Comment:
delete the subsequent installs of black and isort and click

Reply:
done